.buildkite/pipeline.yml

```diff
@@ -25,21 +25,22 @@ steps:
 
   - label: ":golang-lint: Lint"
     key: lint
-    command: rm -rf .git && make lint
+    command: |
+      rm -rf .git && make lint
```

Comment:
Checking out this branch and running `make lint` locally on from my Mac (without `rm -rf .git` beforehand locally, of course) didn't cause any issue…

So I'm wondering if this `rm -rf .git` is still necessary after all (maybe the issues you had with timeout on the tool trying to get vcs info… was from a previous version of `golangci-lint`, before you migrated to the more recent one that might have fixed it since? 🤔

Reply:
Worth trying! I also noticed Buildkite isn't configured for this repo so I'm running builds manually 😅  (I'll create an update to Terraform while at it as well)

Reply:
I was able to reproduce manually by doing the same commands that Buildkite uses for cloning the repo. I think a key difference is that Buildkite [checkouts a detached head to the commit](https://buildkite.com/automattic/hostmgr-packer-plugin/builds/4#01915b35-3e9b-4ed6-8f53-fb07f8b73c89/77-98), as opposed to a specific branch, so when the golang tooling is trying to extract some information about current VCS and likely trying to extract the name of the branch or whatnot, it doesn't handle the case of detached heads well.

Reply:
@iangmaia  I think I found a fix!

Add `GOFLAGS=-buildvcs=false` before invoking the `golangci-lint` command seems to do the trick 🎉

Reply:
> @iangmaia I think I found a fix!

Yay, it worked! 🎉